packages/account/src/Components/reset-trading-password-modal/reset-trading-password-modal.tsx

```diff
@@ -214,7 +214,11 @@ const ResetTradingPassword = ({
                                         </PasswordMeter>
                                     </fieldset>
                                     <Text as='p' size='xs' className='reset-trading-password__hint'>
-                                        <Localize i18n_default_text='Your password must contain between 8-16 characters that include uppercase and lowercase letters, and at least one number and special character such as ( _ @ ? ! / # ).' />
+                                        {platform === CFD_PLATFORMS.DXTRADE ? (
```

Comment:
platform === MT5 is the better approach since we might have future products which will use the password modal.